android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/EncryptionKeyFragment.java

```diff
@@ -132,9 +132,10 @@ private void initState() {
                                         new SpannableStringBuilder();
                                 requestDescription.append(Utils.geteTLD(
                                         encryptionPublicKeyRequest.originInfo.eTldPlusOne));
-                                requestDescription.append(" ");
-                                requestDescription.append(getString(
-                                        R.string.brave_wallet_provide_encryption_key_description));
+                                requestDescription.append(String.format(
+                                        getString(
+                                                R.string.brave_wallet_provide_encryption_key_description),
+                                        ""));
```

Comment:
I think this might not work right if a localizer moves the placeholder to another part of the sentence.

Reply:
changed implementation